src/query.rs

```diff
@@ -10,14 +10,15 @@ pub struct Query;
 #[Object]
 impl Query {
     /// Entity resolver for user of specific id.
-    #[graphql(entity)]
+    //#[graphql(entity)]
```

Comment:
commented out code?

Reply:
Testing stuff that ended up in the repo, will fix.